drivers/docker/config.go

```diff
@@ -553,16 +560,18 @@ type ContainerGCConfig struct {
 }
 
 type DriverConfig struct {
-	Endpoint             string       `codec:"endpoint"`
-	Auth                 AuthConfig   `codec:"auth"`
-	TLS                  TLSConfig    `codec:"tls"`
-	GC                   GCConfig     `codec:"gc"`
-	Volumes              VolumeConfig `codec:"volumes"`
-	AllowPrivileged      bool         `codec:"allow_privileged"`
-	AllowCaps            []string     `codec:"allow_caps"`
-	GPURuntimeName       string       `codec:"nvidia_runtime"`
-	InfraImage           string       `codec:"infra_image"`
-	DisableLogCollection bool         `codec:"disable_log_collection"`
+	Endpoint                    string        `codec:"endpoint"`
+	Auth                        AuthConfig    `codec:"auth"`
+	TLS                         TLSConfig     `codec:"tls"`
+	GC                          GCConfig      `codec:"gc"`
+	Volumes                     VolumeConfig  `codec:"volumes"`
+	AllowPrivileged             bool          `codec:"allow_privileged"`
+	AllowCaps                   []string      `codec:"allow_caps"`
+	GPURuntimeName              string        `codec:"nvidia_runtime"`
+	InfraImage                  string        `codec:"infra_image"`
+	DisableLogCollection        bool          `codec:"disable_log_collection"`
+	PullActivityTimeout         string        `codec:"pull_activity_timeout"`
+	PullActivityTimeoutDuration time.Duration `codec:"-"`
```

Comment:
The parsed duration can be a private field which would ensure that codec ignores it.

```suggestion
	pullActivityTimeoutDuration time.Duration `codec:"-"`
```

Reply:
Awesome, that makes more sense. I'll make that change.